### ISSUE TITLE
[DOCS ISSUE] (newb)

### ISSUE THREAD

**Issue by craycrayultracontrarian** [ipfs/ipfs-docs#1180]
needs ELI5

**Comment by TMoMoreau** [comment]
Hello, we have a basics section in the docs that is designed to be a starting point for new users. You can find that here, https://docs.ipfs.io/concepts/what-is-ipfs/

If you have other IPFS questions that you feel the docs don't answer, you can join the discord server here and ask one of the support channels, https://discord.gg/Pw6gUtCg

If you need general support, you can also check out https://discuss.ipfs.io/

Finally, if you would like to suggest specific new content to be added to the docs, you can open a new issue that includes a brief description of exactly what you are looking for.

Closing this issue, be sure to check out the resources provided and happy learning! :)
